pkgs/top-level/all-packages.nix

```diff
@@ -8702,7 +8702,8 @@ in
     })
     openssl_1_0_1
     openssl_1_0_2
-    openssl_1_1_0;
+    openssl_1_1_0
+    openssl_1_0_2-steam;
```

Comment:
We can add `openssl-steam = openssl_1_0_2-steam;` above and use that name in the steam package.